integration/commands_administration_compat_test.go

```diff
@@ -97,6 +100,67 @@ func TestCommandsAdministrationCompatCollStatsWithScale(t *testing.T) {
 	}
 }
 
+func TestCommandsAdministrationCompatCollStatsCappedCollection(t *testing.T) {
+	t.Skip("https://github.com/FerretDB/FerretDB/issues/2447")
+
+	t.Parallel()
+
+	s := setup.SetupCompatWithOpts(t, &setup.SetupCompatOpts{
+		Providers:                []shareddata.Provider{},
+		AddNonExistentCollection: true,
+	})
+
+	ctx, targetCollection, compatCollection := s.Ctx, s.TargetCollections[0], s.CompatCollections[0]
+
+	for name, tc := range map[string]struct { //nolint:vet // for readability
+		opts *options.CreateCollectionOptions
+	}{
+		"Size": {
+			opts: &options.CreateCollectionOptions{
```

Comment:
`options.CreateCollection().SetCapped`

Please remove all `&options.` in this PR